src/data/matter.ts

```diff
@@ -1,4 +1,62 @@
+import { UnsubscribeFunc } from "home-assistant-js-websocket";
+import { navigate } from "../common/navigate";
 import { HomeAssistant } from "../types";
+import { subscribeDeviceRegistry } from "./device_registry";
+
+export const canCommissionMatterExternal = (hass: HomeAssistant) =>
+  hass.auth.external?.config.canCommissionMatter;
+
+export const startExternalCommissioning = (hass: HomeAssistant) =>
+  hass.auth.external?.fireMessage({
+    type: "matter/commission",
+  });
+
+let CUR_MATTER_DEVICES: Set<string> | undefined;
+let UNSUB_DEVICE_REG: UnsubscribeFunc | undefined;
```

Comment:
Can we move these into the `redirectOnNewMatterDevice` function ? It feels weird to guard against activating this twice, as that would be a bug in our UI.

Reply:
In the dev panel it can activate twice, but we can move the guard to there